cypress/page/address-step.page.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable require-jsdoc */
+class AddressStep {
+    addressCheckout() {
+        return cy.get(
+            ".cart_navigation [type='submit']",
```

Comment:
we expect all page objects have as properties the selectors, and use those properties in the methods. We provide you an example of that in the menu-content.page.ts